.github/workflows/go-lint.yml

```diff
@@ -0,0 +1,25 @@
+name: golangci-lint
+on:
+  pull_request:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  pull-requests: read
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
+          cache: false
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+          only-new-issues: true

```

Comment:
Since this is the first time we introduce the CI lint, we should sweep the code for any existing issues and resolve them. Therefore, we need to set `only-new-issues` to `false`, which is the default value.

Reply:
This issue that caused by Zot V2.0.0-rc.5 update will be resolved in #727

Reply:
Updated. Using the default value to catch all the existing issues